package.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "nhsblocks",
-  "version": "1.2",
```

Comment:
Removed? As long as it does not break the npm dependency and does not need to be published